src/modules/market/components/market-outcomes-list--outcome/market-outcomes-list--outcome.jsx

```diff
@@ -49,9 +48,6 @@ const Outcome = ({
             {outcomeName || (scalarDenomination && scalarDenomination)}{" "}
           </span>
         </div>
```

Comment:
We only want to remove the percentage for scalars, not all market types.

Reply:
oh, good catch! my mistake

Reply:
updated